.github/actions/send-slack-notification/action.yml

```diff
@@ -122,6 +129,7 @@ runs:
         payload: |
           {
             "text": "Notification on ${{ env.EVENT_OUTPUT }} on branch <${{ env.REPO_URL }}/tree/${{ env.BRANCH_NAME }}|`${{ env.BRANCH_NAME }}`>",
+            "thread_ts": "${{ inputs.thread-id }}",
```

Comment:
Did you also check that if this is added to the payload, but empty, it still sends the message correctly, please? (in case `"thread_ts": ""` is not accepted by the rest api (?))

Reply:
Yes, I have. `thread_ts` being empty has no effect on the payload meaning that the message will create a new thread as it did before, this has been tested [here](https://github.com/Alfresco/veracode-report-comparer/actions/runs/8080976505/workflow#L137-L142).